### ISSUE TITLE
haproxy: several bulk deletes not working

### ISSUE THREAD

**Issue by EugenMayer** [opnsense/plugins#1164]
when selecting several items and pressing delete, only one is deleted. Broken at least on
 - backend servers
 - frontend servers

I did not test more, could be more.

**Comment by AdSchellevis** [comment]
for reference, https://github.com/opnsense/core/issues/3062

**Comment by EugenMayer** [comment]
@AdSchellevis seems then to not be (entirely) fixed, happened to me on 19.1 - so maybe something is left / did not get fixed properly

**Comment by fichtner** [comment]
It would require the mutable controller to be used to auto-fix it or add a fix to the controller used in acme-client...

**Comment by AdSchellevis** [comment]
yes, it's not core, but to use the fix, the plugin needs adjustment (preferably use the simpler mutual classes we have now)

**Comment by EugenMayer** [comment]
Thank you!
